test_settings.py

```diff
@@ -68,6 +68,8 @@
     "PARLER_ENABLE_CACHING": False,
     "LANGUAGE_CODE": "en",
     "DEFAULT_AUTO_FIELD": "django.db.models.AutoField",
+    # Due to a recent temporary change in develop-4, we now need to confirm that we intend to use v4
+    "CMS_CONFIRM_VERSION4": True,
```

Comment:
It might be worth changing the base branch we are using for the CMS, so that we aren't bound to develop-4 changes that may come in. 4.0.1.x is a good option.

Reply:
Done